build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.2.0-alpha16'
```

Comment:
Has PR #657 for update 3.2.0-alpha16 to 3.2.1

Reply:
Can I work on this?